src/main/java/Interactors/TextFileTranslator.java

```diff
@@ -4,30 +4,56 @@
 import java.io.FileNotFoundException;
 import java.io.FileWriter;
 import java.io.IOException;
-import java.sql.Array;
 import java.util.ArrayList;
 import java.util.Scanner;
 
 public class TextFileTranslator implements DataAccess{
-    // each subarray holds the instance attributes of a Player instance
-
     private File file;
 
+    /**
+     * Constructor for TextFileTranslator which implements load and save game functionality.
+     * The UseCaseInteractor handles implementation with respect to DataAccessInterface.
+     *
+     * @param file  is the file used to load or save game data to.
+     */
     public TextFileTranslator(File file) {
         this.file = file;
     }
 
+    /**
+     * Searches the given folder denoted by folderPath and returns an array
+     * of Files inside the given folder. Used by UseCaseInteractor to display
+     * a selection of save files to the user when selecting which file to load a game from.
+     *
+     * @param folderPath  is the folder path to the folder which stores the user's save files.
+     * @return  an array of the user's save files.
+     */
+
     public File[] checkSaves(String folderPath){
+        // TODO test in the future, list.Files() may not return as expected for src/save
         // checkSaves searches the given folder and returns an array of file names
         // the user selects the save file from the list of file names
         // loadGame takes in the filePath of the selected file name
         // and returns a String array of Player and Board data
         // the GameCreationInteractor will handle creating the class instances
+
         File folder = new File(folderPath);
         return folder.listFiles();
     }
 
-    @Override
+    /**
```

Comment:
Good job on the javadocs. This would prove to be very helpful in future extensions of adding new features.